src/core/or/command.c

```diff
@@ -480,6 +480,8 @@ command_process_relay_cell(cell_t *cell, channel_t *chan)
   const or_options_t *options = get_options();
   circuit_t *circ;
   int reason, direction;
+  uint32_t orig_delivered_bw = 0;
```

Comment:
Nitpick: Commit msg is wrong, references 27679 instead of 27678.

Reply:
Ok I will fix this in a new branch. I don't like that commit summary anyway, since it doesn't change code.